devel/cram/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                cram
+version             0.7
+categories          devel python
+platforms           darwin
+license             GPL-2+
+maintainers         users.noreply.github.com:SoapZA openmaintainer
```

Comment:
We currently require that you include your github handle. Look for examples in other ports, you will need to use that format. Also, is users.noreply.github.com:SoapZA actually valid for you?